src/felles/domene/elastic/ElasticSearch.ts

```diff
@@ -1,27 +1,67 @@
-import Kandidat from 'felles/domene/kandidat/Kandidat';
-
-export type SearchQuery = {
+export type EsQuery<IndexedItem> = {
```

Comment:
Vi hadde mange ES-typer før, har samlet de til EsQuery og EsResponse. Begge er generiske slik at man setter inn typen for den indekserte tingen der den brukes.